content/docs/reference/microcontrollers/nano-rp2040.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Nano RP2040"
```

Comment:
Arduino Nano RP2040 Connect

Reply:
It probably would have been better to name this board `arduino-nano-rp2040`. All the other boards from arduino have "arduino" in their name. There is no non-connect board so that word is redundant.